### ISSUE TITLE
remove selectize, implement select2

### ISSUE THREAD

**Issue by aristath** [themeum/kirki#1177]

**Comment by aristath** [comment]
After implementing select2 v4, I did some tests and remembered why I had to switch to selectize a year ago...
WooCommerce uses select2 v3.x
As a result, there are conflicts when using Kirki + WooCommerce.

So I tried implementing Select2 v3.x, and did it.
BUT it's a complete mess... completely meaningless code.
I refuse to put something like this in the plugin...

The only way to get drag-n-drop reordering with multiselect dropdowns is to use a hidden input field, and load the options via JS instead of simply creating a <select> element.

Now why the fuck would I do that??? Why would anyone in their right mind accept that compromise?

So...
The choices I've got right now are to either continue to use selectize, or write shitty code to make it work with select2 v 3.5.4 (the version included in WooCommerce).

I think we'll postpone this one and wait until either WordPress core adds select2 (in which case they'll be adding v4.x) or WooCommerce updates select2 to 4.x
